From 576524b4984fc197ffdcc13c7ec92ad86690b572 Mon Sep 17 00:00:00 2001
From: Simon Busch <morphis@gravedo.de>
Date: Thu, 24 Oct 2013 11:21:01 +0200
Subject: [PATCH] maliit-keyboard: disable setting a different surface format

Setting a different surface format leads to EGL_BAD_MATCH once we call eglMakeCurrent so
we have to remove the code part for now.

Upstream-Status: Inappropiate [hardware-dependent]

Signed-off-by: Simon Busch <morphis@gravedo.de>
---
 maliit-keyboard/plugin/inputmethod.cpp | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/maliit-keyboard/plugin/inputmethod.cpp b/maliit-keyboard/plugin/inputmethod.cpp
index 912a3c7..94caaf1 100644
--- a/maliit-keyboard/plugin/inputmethod.cpp
+++ b/maliit-keyboard/plugin/inputmethod.cpp
@@ -78,10 +78,12 @@ namespace {
 
 void makeQuickViewTransparent(QQuickView *view)
 {
+#if 0
     QSurfaceFormat format;
     format.setAlphaBufferSize(8);
     view->setFormat(format);
     view->setColor(QColor(Qt::transparent));
+#endif
 }
 
 QQuickView *getSurface (MAbstractInputMethodHost *host)
-- 
1.8.1.2

